### PR TITLE
HBASE-23698 Switch central repo to https

### PR DESCRIPTION
**Pull request by bharathv**

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 15s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -0 :warning: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ branch-1 Compile Tests _ |
| -1 :x: |  mvninstall  |   0m  9s |  root in branch-1 failed.  |
| -1 :x: |  compile  |   0m  6s |  root in branch-1 failed with JDK v1.8.0_242.  |
| -1 :x: |  compile  |   0m  7s |  root in branch-1 failed with JDK v1.7.0_252.  |
| -1 :x: |  shadedjars  |   0m  5s |  branch has 11 errors when building our shaded downstream artifacts.  |
| -1 :x: |  javadoc  |   0m  6s |  root in branch-1 failed with JDK v1.8.0_242.  |
| -1 :x: |  javadoc  |   0m  7s |  root in branch-1 failed with JDK v1.7.0_252.  |
||| _ Patch Compile Tests _ |
| -1 :x: |  mvninstall  |   0m  7s |  root in the patch failed.  |
| -1 :x: |  compile  |   0m  6s |  root in the patch failed with JDK v1.8.0_242.  |
| -1 :x: |  javac  |   0m  6s |  root in the patch failed with JDK v1.8.0_242.  |
| -1 :x: |  compile  | N/A |  root in the patch failed with JDK v1.7.0_252.  |
| -1 :x: |  javac  | N/A |  root in the patch failed with JDK v1.7.0_252.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| -1 :x: |  xml  |   0m  0s |  The patch has 1 ill-formed XML file(s).  |
| -1 :x: |  shadedjars  |   0m  5s |  patch has 11 errors when building our shaded downstream artifacts.  |
| -1 :x: |  hadoopcheck  |   0m  5s |  The patch causes 11 errors with Hadoop v2.8.5.  |
| -1 :x: |  hadoopcheck  |   0m 10s |  The patch causes 11 errors with Hadoop v2.9.2.  |
| -1 :x: |  javadoc  |   0m  6s |  root in the patch failed with JDK v1.8.0_242.  |
| -1 :x: |  javadoc  |   0m  7s |  root in the patch failed with JDK v1.7.0_252.  |
||| _ Other Tests _ |
| -1 :x: |  unit  |   0m  7s |  root in the patch failed.  |
| +0 :ok: |  asflicense  |   0m  7s |  ASF License check generated no output?  |
|  |   |   3m 40s |   |


| Reason | Tests |
|-------:|:------|
| XML | Parsing Error(s): |
|   | pom.xml |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1053 |
| Optional Tests | dupname asflicense javac javadoc unit shadedjars hadoopcheck xml compile |
| uname | Linux 1aee8547bc1c 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1053/out/precommit/personality/provided.sh |
| git revision | branch-1 / 907184d |
| Default Java | 1.7.0_252 |
| Multi-JDK versions | /usr/lib/jvm/zulu-8-amd64:1.8.0_242 /usr/lib/jvm/zulu-7-amd64:1.7.0_252 |
| mvninstall | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/branch-mvninstall-root.txt |
| compile | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/branch-compile-root-jdk1.8.0_242.txt |
| compile | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/branch-compile-root-jdk1.7.0_252.txt |
| shadedjars | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/branch-shadedjars.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/branch-javadoc-root-jdk1.8.0_242.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/branch-javadoc-root-jdk1.7.0_252.txt |
| mvninstall | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/patch-mvninstall-root.txt |
| compile | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/patch-compile-root-jdk1.8.0_242.txt |
| javac | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/patch-compile-root-jdk1.8.0_242.txt |
| compile | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/patch-compile-root-jdk1.7.0_252.txt |
| javac | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/patch-compile-root-jdk1.7.0_252.txt |
| xml | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/xml.txt |
| shadedjars | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/patch-shadedjars.txt |
| hadoopcheck | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/patch-javac-2.8.5.txt |
| hadoopcheck | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/patch-javac-2.9.2.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/patch-javadoc-root-jdk1.8.0_242.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/patch-javadoc-root-jdk1.7.0_252.txt |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/artifact/out/patch-unit-root.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/testReport/ |
| Max. process+thread count | 41 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/1/console |
| versions | git=1.9.1 maven=3.0.5 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 18s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -0 :warning: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ branch-1 Compile Tests _ |
| -1 :x: |  mvninstall  |   0m  9s |  root in branch-1 failed.  |
| -1 :x: |  compile  |   0m  6s |  root in branch-1 failed with JDK v1.8.0_242.  |
| -1 :x: |  compile  |   0m  8s |  root in branch-1 failed with JDK v1.7.0_252.  |
| -1 :x: |  shadedjars  |   0m  5s |  branch has 11 errors when building our shaded downstream artifacts.  |
| -1 :x: |  javadoc  |   0m  6s |  root in branch-1 failed with JDK v1.8.0_242.  |
| -1 :x: |  javadoc  |   0m  7s |  root in branch-1 failed with JDK v1.7.0_252.  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   8m 29s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 48s |  the patch passed with JDK v1.8.0_242  |
| -1 :x: |  javac  |   1m 48s |  root-jdk1.8.0_242 with JDK v1.8.0_242 generated 38 new + 0 unchanged - 0 fixed = 38 total (was 0)  |
| +1 :green_heart: |  compile  |   1m 55s |  the patch passed with JDK v1.7.0_252  |
| -1 :x: |  javac  |   1m 55s |  root-jdk1.7.0_252 with JDK v1.7.0_252 generated 38 new + 0 unchanged - 0 fixed = 38 total (was 0)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| -1 :x: |  xml  |   0m  0s |  The patch has 1 ill-formed XML file(s).  |
| +1 :green_heart: |  shadedjars  |   3m 14s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |   5m 15s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2.  |
| -1 :x: |  javadoc  |   2m 24s |  root-jdk1.8.0_242 with JDK v1.8.0_242 generated 11 new + 0 unchanged - 0 fixed = 11 total (was 0)  |
| -1 :x: |  javadoc  |   4m 26s |  root-jdk1.7.0_252 with JDK v1.7.0_252 generated 15 new + 0 unchanged - 0 fixed = 15 total (was 0)  |
||| _ Other Tests _ |
| -1 :x: |  unit  | 174m  8s |  root in the patch failed.  |
| +1 :green_heart: |  asflicense  |   0m 32s |  The patch does not generate ASF License warnings.  |
|  |   | 207m 37s |   |


| Reason | Tests |
|-------:|:------|
| XML | Parsing Error(s): |
|   | pom.xml |
| Failed junit tests | hadoop.hbase.mapreduce.TestSecureLoadIncrementalHFiles |
|   | hadoop.hbase.mapreduce.TestLoadIncrementalHFilesUseSecurityEndPoint |
|   | hadoop.hbase.mapreduce.TestLoadIncrementalHFiles |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1053 |
| Optional Tests | dupname asflicense javac javadoc unit shadedjars hadoopcheck xml compile |
| uname | Linux de9ceeaeab7d 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1053/out/precommit/personality/provided.sh |
| git revision | branch-1 / 907184d |
| Default Java | 1.7.0_252 |
| Multi-JDK versions | /usr/lib/jvm/zulu-8-amd64:1.8.0_242 /usr/lib/jvm/zulu-7-amd64:1.7.0_252 |
| mvninstall | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/2/artifact/out/branch-mvninstall-root.txt |
| compile | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/2/artifact/out/branch-compile-root-jdk1.8.0_242.txt |
| compile | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/2/artifact/out/branch-compile-root-jdk1.7.0_252.txt |
| shadedjars | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/2/artifact/out/branch-shadedjars.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/2/artifact/out/branch-javadoc-root-jdk1.8.0_242.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/2/artifact/out/branch-javadoc-root-jdk1.7.0_252.txt |
| javac | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/2/artifact/out/diff-compile-javac-root-jdk1.8.0_242.txt |
| javac | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/2/artifact/out/diff-compile-javac-root-jdk1.7.0_252.txt |
| xml | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/2/artifact/out/xml.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/2/artifact/out/diff-javadoc-javadoc-root-jdk1.8.0_242.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/2/artifact/out/diff-javadoc-javadoc-root-jdk1.7.0_252.txt |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/2/artifact/out/patch-unit-root.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/2/testReport/ |
| Max. process+thread count | 3784 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1053/2/console |
| versions | git=1.9.1 maven=3.0.5 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

File: pom.xml [hunk 1]
```diff
@@ -442,6 +446,14 @@
...
       </snapshots>
     </repository>
+    <repository>
+      <id>central</id>
+      <name>Central Repository</name>
+      <url>https://repo.maven.apache.org/maven2</url>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+    </repository>
```
Comment (saintstack) [inline pom.xml]: I checked the parent apache pom (v22) to see if it had this repo in it. It doesn't. Just has apache repo.

This looks good. When you clean your local repo, are you downloading from https?

Thanks

**Comment by saintstack** [comment]
Its for branch-1 only?

File: pom.xml (hunk 1 above)
Comment (bharathv) [reply pom.xml]: Yes, verified that.

**Comment by petersomogyi** [comment]
I ran into this and thought the build failure was related to HBASE-23675.

**Pull request merged**